### PR TITLE
UNITY-5422 add parameter isZoomEnabled

### PR DESCRIPTION
**Pull request by mhoehn**

### CODE REVIEW

File: apps/data-types.md [hunk 1]
```diff
@@ -256,6 +256,7 @@ This object is used for the attributes of shippingAddress and billingAddress.
 | Attribute | Type | Description |
 | - | - |  - |
 | url | string | The URL of an image. |
+| isZoomEnabled | boolean | True if the product zoom is enabled for an image. |
```
Comment (BirgitBader) [inline apps/data-types.md]: Indicates if the zoom is enabled for a product image.

Comment (mhoehn) [reply apps/data-types.md]: ok

Comment (mhoehn) [reply apps/data-types.md]: response state is now true or false

**Comment by BirgitBader** [comment]
@mhoehn will merge this once you'll let me know about the version it will be in.

**Pull request merged**